module/modmanager/manager.go

```diff
@@ -1083,20 +1077,7 @@ func (m *module) checkReady(ctx context.Context, parentAddr string, logger loggi
 
 // FirstRun is runs a module-specific setup script.
 func (mgr *Manager) FirstRun(ctx context.Context, conf config.Module) error {
```

Comment:
Manager.FirstRun just prepares some high level variables (paths, env). `Module.FirstRun` does most of the heavy lifting now :muscle: